terraform/modules/transfer_throttle/provider.tf

```diff
@@ -0,0 +1,7 @@
+terraform {
```

Comment:
I **think** the provider can be declared once in the top-level `provider.tf`

Reply:
This is because this whole TF operates over two accounts. This allows us to have both accounts at the top level and pass the right one down into each module